### DISCUSSION TITLE
"Search or jump to" keyboard nav doesn't work on macOS with OS keyboard navigation enabled

### DISCUSSION THREAD

**Discussion by daz** [community/community#15383] [Other Feature Feedback, Questions, & Ideas]
macOS Monterey 12.3
Chrome 100

Go to System Preferences, Keyboard, Shortcuts
Check "Use keyboard navigation to move focus between controls"
Go to System Preferences, Accessibility, Keyboard
Check "Enable Full Keyboard Access"
Go to a github.com repo web page
Pressing / or click Search or jump to... box
Up down keys do nothing

**Comment by kimdwkimdw** [comment]
Try another browser. It would be extension problem in Chrome.

**Comment by daz** [comment]
I've updated the title and description of the error. It's accessibility's "Full Keyboard Access" that triggers the error. I tried Safari as well, same bug.
